packages/back-end/src/services/python.ts

```diff
@@ -207,6 +229,67 @@ export const statsServerPool = createPool(
   }
 );
 
+function publishPoolSizeToCloudWatch(value: number) {
+  if (!cloudWatch) return;
+  try {
+    cloudWatch.putMetricData({
+      Namespace: "GrowthBook/PythonStatsPool",
+      MetricData: [
+        {
+          MetricName: "PoolSize",
+          Timestamp: new Date(),
+          Value: value,
+          Unit: "Count",
+          Dimensions: [
+            { Name: "TaskId", Value: process.env.ECS_TASK_ID || "local" },
+          ],
+        },
+      ],
+    });
+  } catch (error) {
+    // When not running on AWS, no need to publish to cloudwatch or warn us every ten seconds.
+  }
+}
+
+function monitorServicePool() {
```

Comment:
question: All of these pool metrics are per-process right?

Do they get tagged automatically with the task or machine identifier?